pom.xml

```diff
@@ -408,15 +414,16 @@
         <plugin>
           <groupId>org.apache.felix</groupId>
           <artifactId>maven-bundle-plugin</artifactId>
-          <version>2.3.7</version>
+          <version>3.5.1</version>
           <extensions>true</extensions>
           <configuration>
             <instructions>
               <Embed-Dependency>*;inline=false;scope=compile</Embed-Dependency>
               <Embed-Transitive>true</Embed-Transitive>
               <Embed-Directory>lib</Embed-Directory>
               <!--Only @Plugin classes in the export packages will be included as plugin-->
-              <_exportcontents>co.cask.bigquery.*,com.google.cloud.hadoop.*,co.cask.gcs.*,co.cask.speech.*,co.cask.common.*</_exportcontents>
+              <_exportcontents>co.cask.bigquery.*,com.google.cloud.hadoop.*,co.cask.gcs.*,
+                co.cask.spanner.*,co.cask.speech.*,co.cask.common.*</_exportcontents>
```

Comment:
this is the package in this repo, not the cask-data/common. we should ideally rename that to avoid this confusion.

Reply:
it doesn't matter where it comes from, this will expose that package and everything under it to every plugin used in the program. It looks like somebody just threw this in here without thinking about it.

The only things that need to be exported are the packages that contain the plugins, and anything that the program needs to run the inputformat/outputformat. This usually means the input format class, output format class, and whatever key and value classes they work with.

Reply:
that package currently has few input format classes, they have to be removed. will do that as a separate PR.